keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandlerTests.kt

```diff
@@ -287,6 +306,51 @@ internal class ClusterHandlerTests : JUnit5Minutests {
           verify { publisher.publishEvent(ofType<ArtifactVersionDeployed>()) }
         }
       }
+
+      derivedContext<SubmittedResource<ClusterSpec>>("exported cluster spec") {
+        deriveFixture {
+          runBlocking {
+            export(exportable)
+          }
+        }
+
+        test("has the expected basic properties") {
+          expectThat(kind)
+            .isEqualTo("cluster")
+          expectThat(apiVersion)
+            .isEqualTo(SPINNAKER_EC2_API_V1)
+          expectThat(spec.regionalIds)
```

Comment:
@emjburns removed the MultiRegion type and regionalIds along with entity tags. Rebasing and removing this assertion will prob clear up the test failure.

Reply:
Yep, I had rebased and didn't notice it broke the test. Fixed.